var/spack/repos/builtin/packages/buildah/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Buildah(MakefilePackage):
```

Comment:
The reason I made this a MakefilePackage is that it autodetects some btrfs dependencies in the Makefile, but not in the regular Go build

Reply:
Do you know how they're doing the autodetection of btrfs dependencies in the makefile? It'd be better to have this as a `GoPackage` so that you get the same level of build isolation as in other Spack builds. Otherwise this is going to install dependencies into `$HOME/go` which isn't what most users want.

Reply:
They check for the existence of the `btrfs/version.h` C++ header in the default search path, and set the `-tags` flag accordingly. Does `GoPackage` have an equivalent feature to set these tags? This would be the `btrfs_noversion exclude_graphdriver_btrfs` tags, probably together with `seccomp exclude_graphdriver_devicemapper` and optionally `libsubid systemd`

Reply:
@alecbcs, do you have an answer for @upsj here?